cfgov/unprocessed/apps/owning-a-home/js/explore-rates/rate-checker.js

```diff
@@ -989,10 +977,12 @@ function init() {
     return false;
   }
 
+  // Record timestamp HTML element that's updated from date from API.
+  timeStampDom = document.querySelector( '#timestamp' );
+
   renderSlider();
   renderChart();
   renderLoanAmount();
-  renderTime();
```

Comment:
This doesn't appear to be needed. Here's the original function:

```
function renderTime( time ) {
  if ( time ) {
    time = formatTime( time );
  }

  $( '#timestamp' ).text( time );
}
```

So if that's called without the `time` argument. The timestamp element would render to `undefined`. Instead I added a default ellipsis to the template and updated the function to:

```
function renderDatestamp( elem, time ) {
  elem.textContent = formatTimestampMMddyyyy( time );
}
```